### PR TITLE
Create first iteration of mentor sign-up form.

### PR DESCRIPTION
**Pull request by Sami-2000**
Write the html for a very simple mentor sign up form where user inputs a single GitHub repo url. Add a button to dashboard page to access this form.

### CODE REVIEW

File: src/main/webapp/mentor_form.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>Sign Up</title>
+<!-- Bootstrap CSS only -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+<!-- Custom styles -->
+<link rel="stylesheet" href="style.css">
+
+<div class="container">
+  <h1 class="display-4 mt-5">Become a Mentor for Your Project!</h1>
+  <h4 class="emphasis mb-5">Enter the url of the GitHub repository for a project that you work on in order to become a mentor.</h4>
```
Comment (Richie78321) [inline src/main/webapp/mentor_form.html:12]: This is a bit wordy, my suggestion isn't final and feel free to mess around with it
```suggestion
  <h4 class="emphasis mb-5">Enter the URL of the GitHub repository for your project and become a mentor.</h4>
```

File: src/main/webapp/mentor_form.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <hr/>
+  <form>
+    <div class="form-group">
+      <label for="inputRepo" class="text-weight-bold">Repository Url</label>
```
Comment (Richie78321) [inline src/main/webapp/mentor_form.html:16]: nit
```suggestion
      <label for="inputRepo" class="text-weight-bold">Repository URL</label>
```

File: src/main/webapp/mentor_form.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <input class="form-control" id="inputRepo" aria-describedby="repoHelp" placeholder="Enter repo url">
```
Comment (Richie78321) [inline src/main/webapp/mentor_form.html:17]: Might be more useful to put an example here
```suggestion
      <input class="form-control" id="inputRepo" aria-describedby="repoHelp" placeholder="https://github.com/octocat/example-repo">
```

File: src/main/webapp/mentor_form.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <small id="repoHelp" class="form-text text-muted">If this project is not already on OpenSesame, we will parse information about it from GitHub for our site.</small>
```
Comment (Richie78321) [inline src/main/webapp/mentor_form.html:18]: Again a nit sorry, but it might be worth emphasizing that we're using _publicly available data_. Something like:
```suggestion
      <small id="repoHelp" class="form-text text-muted">We will use publicly available data about this repository in order to display it on our project search page.</small>
```

Comment (Richie78321) [reply src/main/webapp/mentor_form.html:18]: Could even link to the project search page in this.

File: src/main/webapp/mentor_form.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    </div>
+    <button class="btn text-center">Submit</button>
+  </form>
+</div>
+
+<!-- Add Custom Scripts -->
+<script src="mentors.js"></script>
```
Comment (Richie78321) [inline src/main/webapp/mentor_form.html:25]: nit: no newline

File: src/main/webapp/mentor_form.html (hunk 1 above)
Comment (Sami-2000) [reply src/main/webapp/mentor_form.html:12]: Yah I was playing with this a bit. Like we discussed before, hopefully in the future we will docs with more explicit mentor and mentee guidelines, but love your revision!

File: src/main/webapp/mentor_form.html (hunk 4 above)
Comment (Sami-2000) [reply src/main/webapp/mentor_form.html:18]: Good idea!
